README.md

```diff
@@ -27,9 +35,12 @@ A question we’re often asked is: Doesn’t Kaggle already do this? The central
 
 - **Centralized Leaderboard**: Challenge Organizers whether host their challenge on EvalAI or forked version of EvalAI, they can send the results to main EvalAI server. This helps to build a centralized platform to keep track of different challenges. 
 
-## Goal
```

Comment:
Please don't remove this section.

Reply:
@RishabhJain2018 Should I again add the #Goal section ?

Reply:
yes!

Reply:
@RishabhJain2018  Done ! :+1: